variants/classical/phase.go

```diff
@@ -9,17 +9,25 @@ import (
 	"github.com/zond/godip/variants/classical/orders"
 
 	dip "github.com/zond/godip/common"
+	ord "github.com/zond/godip/orders"
 	cla "github.com/zond/godip/variants/classical/common"
 )
 
-func Phase(year int, season dip.Season, typ dip.PhaseType) dip.Phase {
-	return &phase{year, season, typ}
+func PhaseGenerator(parser ord.Parser) func(int, dip.Season, dip.PhaseType) dip.Phase {
+	return func(year int, season dip.Season, typ dip.PhaseType) dip.Phase {
+		return &phase{year, season, typ, parser}
+	}
+}
+
+func ClassicalPhase(year int, season dip.Season, typ dip.PhaseType) dip.Phase {
```

Comment:
I think the name stutters - I mean, the package is called classical, why have Classical in the type name?

Reply:
Ah, the only objection I had to reusing names was that it made finding usages harder. I was using "find in files" and I've just realised the IDE has a "find usages" feature, so I think renaming this is fine.